### DISCUSSION TITLE
Plugins Not Displaying on "Tools & Plugins" [Bug report]

### DISCUSSION THREAD

**Discussion by sweeet2024** [GetPublii/Publii#1405] [Bug Report]
### Operating system

Windows 11 Pro - OS Build 22621.3155

### Publii version

Version: 0.44.4 (build 16557)

### Issue type

Application

### Bug description

Plugins will not show up in my Toots & Plugins page.  The issue started when the search plugin for a site I'm helping with stopped functioning. I've tried installing the plugin again, I've tried reinstalling Publii several times. 
![Screenshot 2024-02-20 120735](https://github.com/GetPublii/Publii/assets/161523674/d710772e-5bf4-4670-825c-ccde51ef9e4c)
![Screenshot 2024-02-20 120804](https://github.com/GetPublii/Publii/assets/161523674/82932c3b-d071-4d18-9a3b-e801d1bcb29c)
![Screenshot 2024-02-20 120823](https://github.com/GetPublii/Publii/assets/161523674/7b2f724c-2fc4-4f36-a641-e412a1abc86e)
![Screenshot 2024-02-20 120842](https://github.com/GetPublii/Publii/assets/161523674/98dfdfe3-0bba-4ce2-977d-9001bea9cde3)
![Screenshot 2024-02-15 091609](https://github.com/GetPublii/Publii/assets/161523674/1f493ac9-44f8-45e4-a6ab-b4c82930df25)

**Comment by example-user30** [comment]
Several things come to mind - you will need to try some things to rule out some possible causes:
1. You have reinstalled both Publii and the plugin.  Have you rebooted?  I know weird that rebooting a Microsoft OS would fix issues.
2. Maybe there is an issue with Static Search.  Try downloading a different plugin and installing it.  I am using GraphComment - It works in my environment.  I don't own static search, so I have not tried it.
3. Maybe the Publii location that the plugin is being installed to is not where Publii is running from.  Check your "app Settings" - sites location.  You might try moving that to the root of C:\ (C:\publii\sites).
